### ISSUE TITLE
Tabs are not equal widths

### ISSUE THREAD

**Issue by MarkMurphy** [Chion82/native-base-web#3]
If you inspect the Tabs demo you'll notice that the buttons are not equal widths and that the highlight if the middle button is selected, extends over the third tab.

**Comment by Chion82** [comment]
Yeah I confirmed this issue. There are some width calculation problems in the Tab update hooks. Will fix later.

**Comment by RangerMauve** [comment]
Would using flexbox not make more sense for this?

**Comment by Chion82** [comment]
@RangerMauve Worth trying. The tab bars are originally migrated from `react-native-scrollable-tab-view`, where widths are computed by javascript on the fly. I modified some of those code to make it work on browsers but some issues still remain.

**Comment by RangerMauve** [comment]
Try having the tab container be a flexbox with `flex-direction: row` and the tab items have `flex: 1` to make them all take the same width. This approach doesn't work well when you have more tabs than can comfortably fit without any horizontal overflow.

**Comment by MarkMurphy** [comment]
@RangerMauve If your tabs can't fit than you should probably consider another approach or use horizontally scrollable tabs. In general it's probably best to follow [these quidelines](https://material.io/guidelines/components/tabs.html#tabs-types-of-tabs)

Here's some code I wrote yesterday to make my own tabs using flexbox, maybe someone can adapt it: https://gist.github.com/MarkMurphy/3304a78c42b1993abc23801fd99f5caa
